src/Neo/Cryptography/Hasher.cs

```diff
@@ -9,24 +9,21 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
-namespace Neo.SmartContract.Native
+namespace Neo.Cryptography
```

Comment:
I did now understand this change very well in this file.

Reply:
See the updated version of the PR, please.